examples/advanced/extensions/widget.py

```diff
@@ -79,7 +73,6 @@ class IonRangeSlider(InputWidget):
     y = Array.from(x, (x) => Math.pow(x, pow))
```

Comment:
What's up with this code? What is `x` and `y`?

Reply:
oops, missing `const` in front of both